benchmark/SerializerBenchmark/SerializerBenchmark.csproj

```diff
@@ -40,6 +40,13 @@
       <Private>true</Private>
       <SpecificVersion>false</SpecificVersion>
     </Reference>
-  </ItemGroup>
+    <Reference Include="OdinSerializer">
+      <HintPath>OdinSerializer.dll</HintPath>
+      <Aliases>odin</Aliases>
+      <Private>true</Private>
+      <SpecificVersion>false</SpecificVersion>
+      <!-- Odin is not available on NuGet -->
+    </Reference>
```

Comment:
We don't need any of this metadata. Just use `<Reference Include="OdinSerializer.dll" />` and that should be sufficient.
We only use `Aliases` on msgpack because we reference multiple versions of it.